tests/conftest.py

```diff
@@ -15,6 +15,22 @@
 
     Contact: code@inmanta.com
 """
+
+"""
+About the use of @parametrize_any and @slowtest:
+
+For parametrized tests:
+- if the test is fast and tests different things for each parameter, use @parametrize
+- if the parameters only slightly increase the coverage of the test (some different exception path,...), use @parametrize_any.
+- if a test is slow, use @parametrize_any
```

Comment:
If a test is slow, even if the parameters significantly increase the coverage?

Reply:
yes, but this is a tradeoff (as both slow and coverage are vague things)